python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ntp.schema.yml

```
@@ -105,3 +105,33 @@ keys:
         description: List of trusted-keys as string ex. 10-12,15.
         convert_types:
           - int
+      serve:
+        description: Enable Serving NTP to clients
+        type: dict
+        keys:
+          all:
+            type: dict
+            keys:
+              enabled:
+                type: bool
+                required: true
+              access_group:
+                type: str
+                description: Standard ACL to apply to NTP serve. All ACLs are applied in the 'in' direction.
+              ipv6_access_group:
+                type: str
+                description: Standard IPv6 ACL to apply to NTP serve. All ACLs are applied in the 'in' direction.
+              vrfs:
+                type: list
```

This will make it required and unique.
```suggestion
                type: list
                primary_key: name
```